pytorch_lightning/tuner/lr_finder.py

```diff
@@ -366,6 +194,143 @@ def suggestion(self, skip_begin: int = 10, skip_end: int = 1):
             self._optimal_idx = None
 
 
+def lr_find(
```

Comment:
Note to reviewers: This function was moved as we need the `_LRFinder` to be defined